Packs/CadoResponse/Integrations/CadoResponse/CadoResponse.py

```diff
@@ -0,0 +1,493 @@
+            json_data=payload
+        )
+
+
+''' Command Line Handlers '''
+
+
+def test_module(client: Client) -> str:
+    ''' Command handler for !test-module '''
+
+    result: Dict[str, Any] = client.heartbeat()
+    status: Optional[str] = result['status']
+
+    if status is not None and status == 'Running':
+        return 'ok'
+
+    return 'Cado Response is not running'
+
+
+def create_project_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    ''' Command handler for cado-create-project '''
+
+    unix_timestamp: str = str(int(time.time()))
+    project_name: str = args.get('project_name', unix_timestamp)
+    project_description: Optional[str] = args.get('project_description', None)
+    result: Dict[str, Any] = client.create_project(project_name, project_description)
+
+    return CommandResults(
+        outputs_prefix='CadoResponse.Project',
+        outputs_key_field='id',
+        outputs=result
+    )
+
+
+def list_project_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    ''' Command handler for cado-list-project '''
+
+    project_id: Optional[int] = args.get('project_id', None)
+    limit: int = int(args.get('limit', 50))
+
+    if project_id:
+        result: Any = client.get_project(project_id)
+    else:
+        result = client.list_projects(limit)
+
+    return CommandResults(
+        outputs_prefix='CadoResponse.Projects',
+        outputs_key_field='id',
+        outputs=result
+    )
+
+
+def get_pipeline_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    ''' Command handler for cado-get-pipeline '''
+
+    project_id: Optional[int] = args.get('project_id', None)
+    limit: int = int(args.get('limit', 50))
+    pipeline_id: Optional[int] = args.get('pipeline_id', None)
+
+    if pipeline_id:
+        result: CadoResponseCombinedOutput = client.get_pipeline(pipeline_id, project_id)
+    else:
+        result = client.list_pipelines(project_id, limit)
+
+    return CommandResults(
+        outputs_prefix='CadoResponse.Pipelines',
+        outputs_key_field='pipeline_id',
+        outputs=result
+    )
+
+
+def list_ec2_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    ''' Command handler for cado-list-ec2 '''
+
+    project_id: Optional[int] = args.get('project_id', None)
+    region: Optional[str] = args.get('region', None)
+    limit: int = int(args.get('limit', 100))
+    result: List[Dict[str, Any]] = client.list_instances(project_id, region, limit)
+
+    return CommandResults(
+        outputs_prefix='CadoResponse.EC2Instances',
+        outputs_key_field='id',
+        outputs=result
+    )
+
+
+def list_s3_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    ''' Command handler for cado-list-s3 '''
+
+    project_id: Optional[int] = args.get('project_id', None)
+    limit: int = int(args.get('limit', 100))
+    result: Dict[str, Any] = client.list_buckets(project_id, limit)
```

Comment:
if it return 404 error that indicates no data found then please return readable output (if empty list is returned, it should be fine)